src/base/NekRSProblem.C

```diff
@@ -63,6 +63,10 @@ NekRSProblem::NekRSProblem(const InputParameters &params) : NekRSProblemBase(par
 
     if (!_nek_mesh->getMesh().is_replicated())
       mooseError("Distributed mesh features are not yet implemented for moving mesh cases!");
+
+    if (!(_nek_mesh->has_displacements() && _nek_mesh->use_displaced_mesh()))
+      mooseError("Moving mesh problems require displacements and "
```

Comment:
It would be good to have a test that covers this to make sure we have correct usage

Reply:
Thanks! Tests were added in the latest commit `c1d9c6a`.